Code/src/main/java/nl/utwente/viskell/haskell/env/CatalogFunction.java

```diff
@@ -33,6 +37,31 @@
         this.isCommon = isCommon;
     }
 
+    @Override
+    public Map<String, Object> toBundleFragment() {
+        Map<String, Object> bundleFragment = new HashMap<>();
+        bundleFragment.put(Bundleable.KIND, this.getClass().getSimpleName());
+        bundleFragment.put("name", name);
+        bundleFragment.put("signature", signature.toString());
+        bundleFragment.put("category", category);
+        bundleFragment.put("documentation", documentation);
+        bundleFragment.put("isConstructor", isConstructor);
+        bundleFragment.put("isCommon", isCommon);
```

Comment:
AFAIK all FuntionInfos in Environment are CatalogFunctions now, thus the types in Environment could be restricted for now. How to deal with a library of user defined functions is a problem for later.